src/main/scala/no/gdl/bookapi/service/search/IndexBuilderService.scala

```diff
@@ -35,7 +35,7 @@ trait IndexBuilderService {
       synchronized {
         var numIndexed = 0
         val start = System.currentTimeMillis()
-        getLanguages.foreach(language => {
+        translationRepository.allLanguages.foreach(language => {
```

Comment:
Make index for all languages to avoid 404 and reindexing when someone tries to get list of books not published.